documentation/sphinx/source/api-c.rst

```diff
@@ -504,6 +504,57 @@ An |database-blurb1| Modifications to a database are performed via transactions.
 
    Returns a value where 0 indicates that the client is idle and 1 (or larger) indicates that the client is saturated. By default, this value is updated every second.
 
+.. function:: FDBFuture* fdb_database_get_client_status(FDBDatabase* db)
+
+   Returns a JSON string containing database client-side status information. At the top level the report describes the status of the 
+   Multi-Version Client database - its initialization state, the protocol version, the available client versions. The report schema is:
+
+   .. code-block::
+
+      {  "Healthy": <overall health status, true or false>,
+         "InitializationState": <initialization state of the Multi-Version Database>,
```

Comment:
Is it useful and feasible to list the possible states here?

Reply:
Makes sense. I listed the states.